src/data/staticData/extensionData.tsx

```diff
@@ -339,81 +387,126 @@ const extensions: { [key: string]: ExtensionData } = {
     name: MSG.coinMachineName,
     descriptionShort: MSG.coinMachineDescriptionShort,
     descriptionLong: MSG.coinMachineDescriptionLong,
+    descriptionExtended: MSG.coinMachineDescriptionExtended,
+    descriptionLinks: [
+      <ExternalLink
+        text={MSG.coinMachineDescriptionBlogPostLink}
+        href={COIN_MACHINE_BLOG_POST_LINK}
+      />,
+      <ExternalLink
+        text={MSG.coinMachineDescriptionGoogleSheetLink}
+        href={COIN_MACHINE_GOOGLE_SHEET_LINK}
+      />,
+      <ExternalLink
+        text={MSG.coinMachineDescriptionHereLink}
+        href={COIN_MACHINE_DESCRIPTION_LINK}
+      />,
+    ],
     currentVersion: 1,
     createdAt: 1603915271852,
     neededColonyPermissions: [ColonyRole.Root],
-    initializationParams: [
+    tokenContractAddress: (
+      <FormattedMessage
+        {...MSG.coinMachineTokenContractAddress}
+        values={{
+          link: <ExternalLink href={BLOCKSCOUT_LINK} text="Blockscout" />,
```

Comment:
This is not ok.

- If we're on ethereum mainnet this should point to etherescan
- if we're on xdai should point to blockscout
- if on localhost should point to #

Also, it should point directly to the token contract, not just to the general home page. _(that's the point, so the user can actually verify the token contract)_

We actually have a utility that helps in this case: `getBlockExplorerLink`  so you don't have to deal with all the complexity.

Just call it like:
```js
getBlockExplorerLink({
  linkType: 'token',
  addressOrHash: tokenAddress,
});
```

And it will generate the link for you